swan-cern/Dockerfile

```diff
@@ -0,0 +1,94 @@
+ARG VERSION_PARENT=v0.0.1
+
+FROM gitlab-registry.cern.ch/swan/docker-images/jupyter/swan:${VERSION_PARENT}
+
+LABEL maintainer="swan-admins@cern.ch"
+ARG NB_UID="1000"
+ARG BUILD_TAG=daily
+ENV VERSION_DOCKER_IMAGE=$BUILD_TAG
+
+RUN echo "Building swan-cern image with tag ${VERSION_DOCKER_IMAGE} from parent tag ${VERSION_PARENT}."
+
+# Switch to superuser to install packages
+USER root
+
+RUN dnf install -y \
+    # Install Cloudera dependencies - required by IT Spark clusters
+    alsa-lib \
+    at \
+    cronie \
+    cvs \
+    file \
+    gdbm-devel \
+    gettext \
+    jpackage-utils \
+    libXtst \
+    man \
+    passwd \
+    rsyslog \
+    time \
+    xz-lzma-compat \
+    # Required by Oracle
+    libaio && \
+    # Clear the dnf cache as we no longer need to install packages
+    dnf clean all && \
+    rm -rf /var/cache/dnf
+
+# Configure Dask
+# Set Dask environment
+ENV DASK_DIR=/srv/dask
+ENV DASK_LIB_DIR=${DASK_DIR}/lib
+ENV DASK_TLS_DIR=${DASK_DIR}/tls
+
+RUN mkdir -p ${DASK_LIB_DIR} ${DASK_TLS_DIR} && \
+    fix-permissions ${DASK_LIB_DIR} && \
+    chown -R ${NB_USER}:${NB_GID} ${DASK_TLS_DIR}
+
+USER ${NB_UID}
+
+# User session configuration scripts
+# Add scripts to be run before the jupyter server starts
+COPY scripts/before-notebook.d/* /usr/local/bin/before-notebook.d
+
+# dask-labextension is used to disable the server extension automatically
+RUN  pip install --no-deps --no-cache-dir dask-labextension==7.0.0
+
+# Install all of our extensions required to access Spark, HDFS and Dask.
+# Ignore (almost all) dependencies because they have already been installed or come from CVMFS
+RUN pip install --no-deps --no-cache-dir \
+    sparkconnector==2.4.6 \
+    sparkmonitor==2.1.1 \
+    swanportallocator==1.0.1 \
+    swandask==0.0.3
+
+# Install swandaskcluster in its own lib dir so that we can add it individually
+# to the PYTHONPATH of notebooks and terminals, which need it to do automatic
+# TLS configuration for Dask clients
+RUN pip install --no-deps --no-cache-dir --target ${DASK_LIB_DIR} \
+    swandaskcluster==2.0.1
+
+# Add script to create dask secrets
+ADD scripts/others/create_dask_certs.sh /srv/singleuser/create_dask_certs.sh
```

Comment:
`ADD scripts/others/*` ?

Reply:
Done in https://github.com/swan-cern/jupyter-images/pull/19